src/js/modules/artworks.js

```diff
@@ -0,0 +1,72 @@
+import { getArtWorks, addLike } from './ServiceCall.js';
+import { renderModal } from './pupup.js';
+import { getLikesCount } from './counters.js';
+
+const container = document.querySelector('.artworks-list');
+let likeButtons;
+
+const renderArtworks = async () => {
+  let artworkElements = '';
+  container.innerHTML = 'fetching data...';
+  const artworkArray = await getArtWorks().then((result) => result);
+  const likesArray = await getLikesCount();
+
+  artworkArray.data.forEach((artwork) => {
+    let likesCount = 0;
+    let imageLink;
+    if (artwork.image_id !== null) {
+      imageLink = `https://www.artic.edu/iiif/2/${artwork.image_id}/full/843,/0/default.jpg`;
+    } else {
+      imageLink = './images/no-image.png';
+    }
+    const likes = likesArray.filter(
+      /* eslint-disable */
+      (element) => element.item_id == artwork.id
+      /* eslint-enable */
```

Comment:
- [ ] Please do not disable linters instead kindly find a solution to the error instead of disabling the linter. Having linter checks helps us to write better code. Try checking StackOverflow for a possible solution.